librz/analysis/class.c

```diff
@@ -53,6 +53,25 @@ static const char *attr_type_id(RzAnalysisClassAttrType attr_type) {
 	}
 }
 
+RZ_API void rz_analysis_class_recover_from_rzbin(RzAnalysis *analysis) {
+	rz_cons_break_push(NULL, NULL);
+	RzList *classes = rz_bin_get_classes(analysis->binb.bin);
+	if (classes) {
+		RzListIter *iter_class;
+		RzBinClass *class;
+		rz_list_foreach (classes, iter_class, class) {
+			if (!rz_analysis_class_exists(analysis, class->name)) {
```

Comment:
If you use `rz_cons_break_push()`, you should also check `rz_cons_is_breaked()` in every iterationa and break if it returns true.

Reply:
okay, I see.